pkg/apis/storage/validation/validation.go

```diff
@@ -473,6 +475,44 @@ func validateFSGroupPolicy(fsGroupPolicy *storage.FSGroupPolicy, fldPath *field.
 	return allErrs
 }
 
+// validateServiceAccountTokens tests if the Audience in each ServiceAccountToken are different and not empty.
+// Besides, at most one ServiceAccountToken can ignore Audience.
+func validateServiceAccountTokens(serviceAccountTokens []storage.ServiceAccountToken, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if serviceAccountTokens != nil && utilfeature.DefaultFeatureGate.Enabled(features.CSIServiceAccountToken) {
+		audiences := make(map[string]bool)
+		for _, serviceAccountToken := range serviceAccountTokens {
+			audience := ""
+			if serviceAccountToken.Audience != nil {
+				audience = *serviceAccountToken.Audience
+				if audience == "" {
+					allErrs = append(allErrs, field.Invalid(fldPath, audience, "empty string is not a valid audience"))
```

Comment:
> empty string is not a valid audience

Doesn't "" mean audience is the kube-apiserver elsewhere?

Reply:
right. empty string is valid actually. done. audience is now required as it is gonna be the key for strategic merge